Extensions/Today/TodayViewController.swift

```diff
@@ -38,11 +41,14 @@ class TodayViewController: UIViewController, NCWidgetProviding {
         imageButton.label.text = TodayStrings.NewTabButtonLabel
 
         let button = imageButton.button
-
-        button.setImage(UIImage(named: "new_tab_button_normal")?.withRenderingMode(.alwaysTemplate), for: .normal)
-        button.setImage(UIImage(named: "new_tab_button_highlight")?.withRenderingMode(.alwaysTemplate), for: .highlighted)
-
+        button.frame = CGRect(width: 60.0, height: 60.0)
```

Comment:
Is this tested on smaller and larger devices, does it look fine on iPad, iPhone5, iPhone 11 Pro Max?

Reply:
It's tested on all devices and it looked fine.